src/events/onPluginInstalledEmitter.ts

```diff
@@ -0,0 +1,6 @@
+import { EventEmitter } from "vscode";
+
+const onPluginInstalledEmitter = new EventEmitter<void>();
```

Comment:
why like this and not `exprot default const`?

Reply:
because default export is shit 🤮